lib/skate/settings/user.ex

```diff
@@ -20,6 +20,19 @@ defmodule Skate.Settings.User do
     Skate.Repo.get_by(DbUser, email: String.downcase(email))
   end
 
+  @spec get_by_id(integer()) :: DbUser.t() | nil
```

Comment:
👍 assuming this will land before https://github.com/mbta/skate/pull/1789, I'll modify it to reduce duplication by using this new `get_by_id`

Reply:
Whatever works for you! I was just looking at #1789 and thinking about how I'd rebase this PR on top of it if I needed to anyway.